pkg/metrics/series.go

```diff
@@ -93,6 +94,40 @@ func (series Series) Marshal() ([]byte, error) {
 	return proto.Marshal(payload)
 }
 
+// MarshalStrings converts the timeseries to a sorted slice of string slices
+func (series Series) MarshalStrings() ([]string, [][]string) {
+	var headers = []string{"Metric", "Type", "Timestamp", "Value", "Tags"}
+	var payload = make([][]string, len(series))
+
+	for _, serie := range series {
+		payload = append(payload, []string{
+			serie.Name,
+			fmt.Sprintf(serie.MType.String()),
+			fmt.Sprintf("%.0f", serie.Points[0].Ts),
+			fmt.Sprint(serie.Points[0].Value),
```

Comment:
Same here: do we need Sprintf

Reply:
What would be an alternative to convert a float64 value into a string?  Leaving it as `Value` results in an error for the string slice. 

(New to Go so could be missing something obvious!)

Reply:
May help: https://golang.org/pkg/strconv/

Reply:
Looks like `strconv.FormatFloat(serie.Points[0].Value, 'f', -1, 64)` is about 40% faster.